trackintel/preprocessing/triplegs.py

```diff
@@ -86,11 +86,13 @@ def generate_trips(spts, tpls, gap_threshold=15):
     # If the triplegs already have a column "trip_id", we drop it
     if "trip_id" in tpls:
         tpls.drop(columns="trip_id", inplace=True)
+        #todo: should we add a warning here?
```

Comment:
If we would alter it in place I would be totally for it. But when returning a copy is that really necessary?

Reply:
👍